packages/elements/src/react/common/form/index.tsx

```diff
@@ -40,22 +40,24 @@ const useFieldContext = () => useContext(FieldContext);
  */
 const useForm = ({ flowActor }: { flowActor?: BaseActorRef<{ type: 'SUBMIT' }> }) => {
   const error = useFormSelector(globalErrorsSelector);
+  const [isPending, startTransition] = useTransition();
```

Comment:
Testing using React's native transitions. Not sure if it provides value beyond the loading state tracking within the state machine.